### PR TITLE
Implement websocket on student delete

### LINKED ISSUE #472: Student delete websocket

### PR DESCRIPTION
**Pull request by victormylle**
Closes #472

### CODE REVIEW

File: frontend_nextjs/utils/ApiClient.js [hunk 1]
```diff
@@ -115,7 +115,7 @@ export class Url {
 
                 return { success: true, data: res }
             }
-            const resp = await axios.get(this._url, { "headers": this._headers});
+            const resp = await axios.get(this._url, { "headers": this._headers });
             return { success: true, data: resp.data };
         } catch (e) {
             console.log(e)
```
Comment (StefVdHaute) [inline frontend_nextjs/utils/ApiClient.js:121]: logging statement

**Review by StefVdHaute — CHANGES_REQUESTED** [review]
Functionally it works but the user experience could be improved:
- When confirming the delete nothing changes and it seems nothing happened.
- When i click away and i go back, the detail screen is loading as you implemented but the user is still in the list until i reload.

**Comment by victormylle** [comment]
It works when I try it, your websockets doesn't seem to work. Maybe refresh or delete the cache and cookies.

**Comment by StefVdHaute** [comment]
My bad it worked perfectly without old cookies and cache.

**Review by StefVdHaute — APPROVED** [review]

**Pull request merged**